seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseFileSinkConfig.java

```diff
@@ -71,6 +72,10 @@ public BaseFileSinkConfig(@NonNull Config config) {
         }
         checkNotNull(path);
 
+        if (path.equals(File.separator)) {
+            this.path = "";
+        }
```

Comment:
what's this use for?

Reply:
https://github.com/apache/seatunnel/issues/6077 
<img width="374" alt="image" src="https://github.com/apache/seatunnel/assets/41814775/356d7599-b218-4117-bec4-ae7e78c9f779">

When the path is the root path, the concatenated temporary directory information will become "//tmp" , task failed



java.lang.NullPointerException: null
        at org.apache.seatunnel.transform.sql.SQLTransform.close(SQLTransform.java:188) ~[seatunnel-transforms-v2.jar:2.3.4-SNAPSHOT]
        at org.apache.seatunnel.engine.server.task.flow.TransformFlowLifeCycle.close(TransformFlowLifeCycle.java:119) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at org.apache.seatunnel.engine.server.task.SeaTunnelTask.lambda$close$5(SeaTunnelTask.java:330) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:?]
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655) ~[?:?]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?]
        at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:290) ~[?:?]
        at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doInvoke(ForkJoinTask.java:408) ~[?:?]
        at java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:736) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:159) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:173) ~[?:?]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233) ~[?:?]
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497) ~[?:?]
        at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:661) ~[?:?]
        at org.apache.seatunnel.engine.server.task.SeaTunnelTask.close(SeaTunnelTask.java:327) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at org.apache.seatunnel.engine.server.TaskExecutionService$BlockingWorker.run(TaskExecutionService.java:651) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]

Reply:
oh, I see. Could you add a test case to cover this case? Thanks.

Reply:
ok, completed , Thanks.

Reply:
@TyrantLucifer PTAL.